lib/paratii.eth.wallet.js

```diff
@@ -15,21 +15,31 @@ export function patchWallet (wallet, config) {
       if (this._mnemonic) {
         mnemonic = this._mnemonic
       } else {
-        mnemonic = newMnemonic()
+        mnemonic = bip39.generateMnemonic()
       }
     }
-    if (isValidMnemonic(mnemonic)) {
+    if (bip39.validateMnemonic(mnemonic)) {
       this._mnemonic = mnemonic
       this._hdIndex = 0
-      // this code is lifted from eth-lightwallet
-      var hdRoot = new Mnemonic(this._mnemonic).toHDPrivateKey().xprivkey
+
+      var seed = bip39.mnemonicToSeed(this._mnemonic, this._passphrase)
+      // saving for testing purpose
```

Comment:
There are two reasons why this is not good:
* as a general rule, we should avoid having code only needed for testing in the library. 
* security: the seed (And the privatekeys) are the secrets here, and are to be encrypted with the passphrase. Saving them as properties on the wallet sort of defeats all of this.

Reply:
(I realize we do the same with the `_mnemonic` property, which has the same risks. We probably should change that as well. Please let me know ifyou change this, because that will perhaps break things in the portal as well)